Microsoft.Toolkit.Uwp/Helpers/DispatcherHelper.cs

```diff
@@ -204,17 +188,35 @@ public static Task<T> AwaitableRunAsync<T>(this CoreDispatcher dispatcher, Func<
         /// <returns>Awaitable Task</returns>
         public static Task AwaitableRunAsync(this CoreDispatcher dispatcher, Func<Task> function, CoreDispatcherPriority priority = CoreDispatcherPriority.Normal)
         {
-            return dispatcher.AwaitableRunAsync<object>(
```

Comment:
Okay, so i understand that you decide to make each `AwaitableRunAsync` to deal with the type of `function` in their own way without relying on the `Func<Task<T>>` and `Func<T>` overload to minimize the amount of lambda we have to create. But then again, these are overlapping code and basically we are maintaining 4 different copies of the relatively same piece of code. I don't think having them separated is a good idea imo.